source/kaleidic/lubeck.d

```diff
@@ -1031,6 +1031,11 @@ unittest
     assert (det(ssing) == 0);
     assert (detSymmetric('L', ssing) == 0);
 
+    // check determinant of empty matrix
+    assert(slice!double(0, 0).det == 1);
+    // check determinant of zero matrix
+    assert(repeat(0, 9).sliced(3, 3).det == 0);
```

Comment:
btw, `repeat(0, 9).sliced(3, 3).` can be just `repeat(0, 3, 3)` or `repeat(0, [3, 3])`